### ISSUE TITLE
Detailed results for estimates widget all show "ERROR: Rendered has not been set"

### ISSUE THREAD

**Issue by swernli** [microsoft/qsharp#1880]
When using the widgets to display a resource estimation result, if the checkbox for "Show detailed rows" is selected, then each of the detailed explanations' text start with "ERROR: Rendered has not been set." Not sure where exactly that's coming from, but I wonder if it is a side of effect of trying to display LaTex in the paragraphs?

<img width="1126" alt="image" src="https://github.com/user-attachments/assets/833019e0-def0-48d7-bc24-ed2399c6bf10">

**Comment by idavis** [comment]
There error is typo. The renderer has no been set. See `npm/qsharp/ux/rederers.tsx`. When working in notebooks we need to ensure the default renderer is replaced with a call to `setRenderer`.

```ts
// Default renderer to be replaced before components are first used.
// Expectation is that this will convert markdown and/or LaTeX to HTML.
let theRenderer = function (input: string): string {
  const err = "ERROR: Rendered has not been set";
  console.error(err);
  return err + ". " + input;
};

export function setRenderer(renderer: (input: string) => string) {
  theRenderer = renderer;
}
```

**Comment by billti** [comment]
Fixed in #1957
